projects/plugins/jetpack/jetpack.php

```diff
@@ -169,7 +169,7 @@ function jetpack_admin_missing_files() {
 				printf(
 					wp_kses(
 						/* translators: Placeholder is a link to a support document. */
-						__( 'Your installation of Jetpack is incomplete. If you installed Jetpack from GitHub, please refer to <a href="%1$s" target="_blank" rel="noopener noreferrer">this document</a> to set up your development environment. Jetpack must have Composer dependencies installed and built via the build command.', 'jetpack' ),
+						__( 'Your installation of Jetpack is incomplete. If you installed Jetpack from GitHub, please refer to <a href="%1$s" target="_blank" rel="noopener noreferrer">this document</a> to set up your development environment. Jetpack must have Composer dependencies installed and built via the build command: <code>jetpack build plugins/jetpack --with-deps</code>', 'jetpack' ),
```

Comment:
Since you're adding `code`, could you add it to the array of allowed tags below?

Reply:
[bb4f2ab](https://github.com/Automattic/jetpack/pull/33414/commits/bb4f2ab7ae9c0fb02b83f018f73a715cfb4b35c3)